src/components/Modal/useModal.tsx

```diff
@@ -0,0 +1,10 @@
+import { useState } from 'react';
+
+export const useModal = () => {
+  const [isOpen, SetIsOpen] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isOpen, setIsOpen] = useState<boolean>(false);
```
tiny nitpick 😬 I think we always use lowercase letter for const and function, only components that starts with uppercase letter